src/workers/workerHelpers.ts

```diff
@@ -322,6 +328,53 @@ function validateEnum(value: string, options: string[] | null): ValidationReturn
     : false;
 }
 
+/**
+ * Convert a repeat argument to a valid json and calculate the size of a repeated argument.
+ *
+ * @param {string} str - The string representation of the repeated argument.
+ * @returns {number} The size of the repeated argument.
+ */
+function repeatSize(str: string): number {
+  // Remove whitespace and newlines
+  str = str.replace(/\s/g, '');
+
+  // Add quotes to property names
+  str = str.replace(/(\w+)(?=:)/g, '"$1"');
+
+  // add quotes to values
+  str = str.replace(/(?<=:)\b(\w+)\b/g, '"$1"');
+
+  // Assign default values to properties with missing values
+  str = str.replace(/:\s*(?=[\]}])/g, ': ""');
+
+  // Remove trailing commas
+  str = str.replace(/,\s*}/g, '}').replace(/,\s*]/g, ']');
+
+  return (JSON.parse(str) as any[]).length;
+}
+
+/**
+ * Validates the repeat count of a value based on the expected argument configuration.
+ *
+ * @param {string} value - The value to validate.
+ * @param {FswCommandArgumentRepeat} expected_argument - The expected argument configuration defining the repeat count constraints.
+ * @returns {ValidationReturn} Either a custom error code or `false` indicating a valid repeat count.
+ */
+function validateRepeat(value: string, expected_argument: FswCommandArgumentRepeat): ValidationReturn {
+  // no range info, ignore check
+  if (expected_argument.repeat.min === undefined && expected_argument.repeat.max === undefined) {
+    return false;
+  }
+
+  const numberOfRepeats = repeatSize(value);
+  const min = expected_argument.repeat.min ? expected_argument.repeat.min : 0;
+  const max = expected_argument.repeat.max ? expected_argument.repeat.max : Infinity;
+
+  return numberOfRepeats < min || numberOfRepeats > max
+    ? CustomErrorCodes.InvalidArgumentCount(numberOfRepeats, min, max)
```

Comment:
I am not seeing this error in the editor anymore after the latest changes (for either < min or > max repeat args).
In fact I don't think any errors are coming through?

Possible related, seeing this error in the dev console if I type in the editor:

```
tsMode.js:393 Uncaught (in promise) TypeError: yt is not iterable
    at hat (customTS.worker.js:322:3504)
    at eEe (customTS.worker.js:322:3863)
    at aw.getSemanticDiagnostics (customTS.worker.js:322:6172)
```

Reply:
Noting I tried manually re-building the custom worker but still get the same error.

Reply:
let me take a look

Reply:
Fixed in https://github.com/NASA-AMMOS/aerie-ui/pull/760